libethash-cuda/CMakeLists.txt

```
@@ -28,8 +28,8 @@ else()
 	endif()
 endif()
 
-file(GLOB sources "*.cpp" "*.cu")
-file(GLOB headers "*.h" "*.cuh")
+file(GLOB sources "dag_generation_kernel.cu" "ethash/ethash_cuda_miner_kernel.cu" "ethash/CUDAMiner.cpp")
```

But without `GLOB` :)

```suggestion
set(sources
    dag_generation_kernel.cu
    ethash/ethash_cuda_miner_kernel.cu
    ethash/CUDAMiner.cpp
    CUDAMiner.h
    ethash/ethash_cuda_miner_kernel.h
)
```

---

Will change on other branch :)